global/meta/models.yml

```diff
@@ -452,6 +429,38 @@ user:
     required: True
     restriction_mode: F
 
+meeting_user:
+  id:
+    type: number
+    required: true
+    restriction_mode: A
+  user_id:
+    type: relation
+    to: user/meeting_user_ids
+    required: true
+    restriction_mode: A
+  meeting_id:
+    type: relation
+    to: meeting/meeting_user_ids
+    required: true
+    restriction_mode: A
+  comment:
+    type: HTMLStrict
+    restriction_mode: A
+  number:
+    type: string
+    restriction_mode: A
+  structure_level:
+    type: string
+    restriction_mode: A
+  about_me:
+    type: HTMLStrict
+    restriction_mode: A
```

Comment:
Set to description_mode B and insert a `description: "restriction_mode B is restriction_mode A or request user == user_id"`

Reply:
I don't think the description of the restriction mode should be part of the `models.yml`. We already have these in the AU service and neither doubling the information nor becoming inconsistent with other models/restriction modes provides any value.

Reply:
Set restriction_mode(!) to B. As @jsangmeister said, I don't include a description.

Reply:
This is a restriction mode we invented. If you only write "B" neither Oskar  and in some weeks nor we know what restriction mode B should be. If you have a better place to describe this for Oskar then document it there!.